fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -344,6 +344,35 @@ public class Config extends ConfigBase {
      */
     @ConfField public static int http_port = 8030;
 
+    /**
+     * ssl key store path.
+     * If you want to change the path, you need to create corresponding directory.
+     */
+    @ConfField public static String key_store_path = System.getenv("DORIS_HOME")
+        + "/conf/ssl/doris_ssl_certificate.keystore";
+
+    /**
+     * ssl key store password. Null by default.
+     */
+    @ConfField public static String key_store_password = "";
+
+    /**
+     * ssl key store type. "JKS" by default.
+     */
+    @ConfField public static String key_store_type = "JKS";
+
+    /**
+     * ssl key store alias. "doris_ssl_certificate" by default.
+     */
+    @ConfField public static String key_store_alias = "doris_ssl_certificate";
+
+    /**
+     * ssl enable flag. false by default.
+     * If the value is false, http is supported. Otherwise, https is supported.
+     * Currently doris uses many ports, so http and https are not supported at the same time.
+     */
+    @ConfField public static boolean ssl_enable = false;
```

Comment:
Please update the document and describe how to use these config
https://doris.apache.org/zh-CN/docs/dev/admin-manual/config/fe-config

Reply:
ok